### ISSUE TITLE
Cannot read ID3V1 tag on unicode system (wrong encoding)

### ISSUE THREAD

**Issue by maazl** [MediaArea/MediaInfoLib#1265]
If the local system uses UTF8 encoding (common on many Linux variants) MediaInfoLib cannot read ID3V1 tags containing 8 bit characters anymore. The fields are simply missing.

Looking at File_Id3.cpp it seems that it tries to read the tags with the local encoding. But UTF-8 was never common for ID3V1. Mostly ISO-8859-1 was used (same as the default of old ID3V2 tags). But other e.g russian codepages were used sometimes too.
Switching the locale to ISO-8859-1 is no option either because this raises several other problems with file names and of course the output.

I think either ID3V1 should be read as ISO-8859-1 (most common) or there should be an option to override the encoding when reading ID3V1 (and maybe other non-unicode meta info).

Example:
```
0023ABFC 55 55 55 55 │ 55 55 55 55 │ 55 55 55 55 │ 55 55 55 55 │ 55 55 55 55 │ 55 55 55 55 │ 55 55 55 54  UUUUUUUUUUUUUUUUUUUUUUUUUUUT
0023AC18 41 47 4A 75 │ 6C 65 62 75 │ 6B 6B 20 66 │ E4 6C 6C 74 │ 20 76 6F 6D │ 20 48 69 6D │ 6D 65 6C 00  AGJulebukk f.llt vom Himmel.
0023AC34 00 00 00 00 │ 43 6F 72 6E │ 65 6C 69 61 │ 20 46 75 6E │ 6B 65 00 00 │ 00 00 00 00 │ 00 00 00 00  ....Cornelia Funke..........
0023AC50 00 00 00 00 │ 00 00 57 65 │ 69 68 6E 61 │ 63 68 74 73 │ 6D 61 6E 6E │ 20 43 44 31 │ 00 00 00 00  ......Weihnachtsmann CD1....
0023AC6C 00 00 00 00 │ 00 00 00 00 │ 32 30 30 35 │ 43 44 00 00 │ 00 00 00 00 │ 00 00 00 00 │ 00 00 00 00  ........2005CD..............
0023AC88 00 00 00 00 │ 00 00 00 00 │ 00 00 00 00 │ 00 01 B7                                               ...............
```
The E4 character prevents reading of the title tag. It is the ISO-8851-1 code of 'ä'. (Several other 8 bit characters have the same effect.)

**Comment by JeromeMartinez** [comment]
True, in practice nowadays non-ASCII letters are handled only on Windows for ID3v1.
I think that adding an option for forcing ISO-8859-1 would be fine and not difficult.

I am reluctant to add more code pages, I would have to map them and I am not sure it is useful for a lot of people. ISO-8859-1 is already implemented (due to ID3v2) so even less difficult.
